staging/src/k8s.io/apiserver/pkg/endpoints/handlers/apply.go

```diff
@@ -96,22 +82,7 @@ func (p *applyPatcher) saveNewIntent(patch map[string]interface{}, workflow stri
 	// Make sure we have the gvk set on the object.
 	(&unstructured.Unstructured{Object: patch}).SetGroupVersionKind(p.kind)
```

Comment:
@apelisse I am not sure why this is explicitly required here and if it is required for the new intent as well.
If yes, I would move it to intent.go and remove applyPatcher.saveNewIntent entirely.

Reply:
Ah, okay.
Then I'll move it